.semaphore/generate-semaphore-yaml.sh

```diff
@@ -1,9 +1,15 @@
-#!/usr/bin/env bash
+#!/bin/bash
 
 for out_file in semaphore.yml semaphore-scheduled-builds.yml; do
-  echo "# !! WARNING, DO NOT EDIT !! This file is generated from semaphore.yml.tpl." > $out_file
-  echo "# To update, modify the template and then run 'make gen-semaphore-yaml'." >> $out_file
-  cat semaphore.yml.tpl >> $out_file
+  echo "# !! WARNING, DO NOT EDIT !! This file is generated from semaphore.yml.tpl." >$out_file
+  echo "# To update, modify the template and then run 'make gen-semaphore-yaml'." >>$out_file
+
+  cat semaphore.yml.d/01-preample.yml >>$out_file
```

Comment:
language nit: preample -> preamble (should also rename the file)

Reply:
Fixed.